terraform/main.tf

```diff
@@ -256,6 +256,51 @@ resource "google_cloudbuild_trigger" "build-and-publish" {
   }
 }
 
+locals {
+  common_cloudrun_env_vars = [
+    {
+      name  = "DB_POOL_MIN_CONNS"
+      value = "2"
+    },
+    {
+      name  = "DB_POOL_MAX_CONNS"
+      value = "10"
+    },
+    {
+      name  = "DB_PASSWORD_SECRET"
+      value = google_secret_manager_secret_version.db-pwd-initial.name
```

Comment:
We changed this a bit. Secrets no longer need the `_SECRET` suffix and should start with `secret://` instead:

```terraform
name = "DB_PASSWORD"
value = "secret://${google_secret_manager...}"
```

Reply:
Thanks, fixed.